benchmarks/newton_solver_benchmark_set/spiegelman_et_al_2016/input.prm

```diff
@@ -0,0 +1,140 @@
+set Additional shared libraries            = ./libdrucker_prager_compositions.so
```

Comment:
Can you alight `=` signs and values within subsections for readability?

Reply:
done :)